src/state/state.ts

```diff
@@ -41,8 +32,8 @@ export interface State {
   gasPriceStore: Record<string, Record<string, GasState>>;
   derivedSponsorWallets: Record<DapiName, PrivateKey>;
   signedApiStore: Record<DataFeedId, SignedData>;
-  signedApiUrlStore: Record<ChainId, Record<Provider, string[]>>;
-  dapis: Record<DApiName, DapiState>;
+  signedApiUrlStore: Record<ChainId, Record<ProviderName, string[]>>;
```

Comment:
Should we also give a distinct name to the `SignedApiUrl` type here?
```suggestion
  signedApiUrlStore: Record<ChainId, Record<ProviderName, SignedApiUrl[]>>;
```